Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -874,6 +874,34 @@ i32 HTMLInputElement::default_tab_index_value() const
     return 0;
 }
 
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-checkvalidity
+WebIDL::ExceptionOr<bool> HTMLInputElement::check_validity()
+{
+    dbgln("(STUBBED) HTMLInputElement::check_validity(). Called on: {}", debug_description());
+    return true;
+}
+
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-reportvalidity
+WebIDL::ExceptionOr<bool> HTMLInputElement::report_validity()
+{
+    dbgln("(STUBBED) HTMLInputElement::report_validity(). Called on: {}", debug_description());
+    return true;
+}
+
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#dom-cva-setcustomvalidity
+WebIDL::ExceptionOr<void> HTMLInputElement::set_custom_validity(DeprecatedString const& error)
```

Comment:
This ("must set the custom validity error message to error.") cannot possibly throw :)

Reply:
Right, thanks! Same with HTMLInputElement.select() looks like, I'll fix it too.